CMakeLists.txt

```diff
@@ -365,7 +365,7 @@ else()
   foreach(_cxxflag  -Werror -Wall -Wextra -Wpointer-arith -Winvalid-pch -Wcast-align
                     -Wwrite-strings -Woverloaded-virtual -Wnon-virtual-dtor -Wold-style-cast
                     -Wstrict-null-sentinel -Wsign-promo -fdiagnostics-show-option
-                    -fstack-protector-all)
+                    -fstack-protector-all -Wimplicit-fallthrough=0)
```

Comment:
I would prefer to not disable this warning for the whole project. We already have comments in our own code to signal to GCC that a fall through is intentional. I suggest to patch jsoncpp instead, see https://github.com/open-source-parsers/jsoncpp/pull/697 for an easy fix.

Reply:
Ok. i'll patch jsoncpp.cpp and submit.